SimCalorimetry/EcalTrigPrimProducers/plugins/EcalTrigPrimESProducer.h

```diff
@@ -40,6 +43,9 @@
 #include "CondFormats/EcalObjects/interface/EcalTPGTowerStatus.h"
 #include "CondFormats/EcalObjects/interface/EcalTPGWeightGroup.h"
 #include "CondFormats/EcalObjects/interface/EcalTPGWeightIdMap.h"
+#include "CondFormats/EcalObjects/interface/EcalTPGOddWeightGroup.h"
+#include "CondFormats/EcalObjects/interface/EcalTPGOddWeightIdMap.h"
+#include "CondFormats/EcalObjects/interface/EcalTPGTPMode.h"
 
 #include "zlib.h"
 
```

Comment:
I guess that is beyond this PR but at one point in the future: header files are advised to be merged into the cc file, unless they are use by other modules (in that case they should go to /interface/